public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -173,6 +211,15 @@ define(function(require) {
       buttons.parent.removeClass("hide");
 
       publisher.updateDialog("");
+
+      Project.publishNeedsUpdate(false, function(err) {
+        if(err) {
+          console.error("[Bramble] Failed to set the publishNeedsUpdate flag after unpublishing with: ", err);
+          return;
```

Comment:
Sometimes you `console.error` and then return, other times you let it run through.  Should you always do the same?